src/main/java/org/jenkinsci/backend/cifiller/App.java

```diff
@@ -97,6 +97,10 @@ public void ensure(GHRepository r) throws IOException {
         } catch (FileNotFoundException fnfe) {
             System.err.printf("WARN: Missing permission to access to hooks: %s\n", r.getName());
         }
+        if (cli.execute(Arrays.asList("build",jobName),
+                new NullInputStream(0),new NullOutputStream(),new NullOutputStream())==0) {
+            throw new Error("Initial build failed");
```

Comment:
I'm not sure @slide I would like to kill the loop if the initial build fails. WDYT ?
For now I tried to never break the loop and to display logs
The disadvantage is that it makes difficult to see if something goes wrong (note that we have no notification for now - I may add some ...)
The advantage is that it allows to continue to process others repos/jobs if one is failing